docs/book/driver-deployment/installation.md

```diff
@@ -80,6 +81,8 @@ Where the entries have the following meaning:
 
 - `ca-file` - path to a CA certificate in PEM format. It is an optional parameter.
 
+- `Thumprint` - the certificate thumprint. It is an optional parameter and is ignored when you're using insesure setups or when you provide `ca-file`
```

Comment:
`Thumprint` -> `Thumbprint`
the certificate thumprint -> the certificate thumbprint

insesure -> insecure